### ISSUE TITLE
Expand on Primary Host caveats

### ISSUE THREAD

**Issue by wes-johnson** [eclipse/sparkplug#12]

**Comment by I-Campbell** [comment]
I have seen here:
https://github.com/eclipse/sparkplug/blob/develop/docs/normative_statements.md#sparkplug-primary-host-client
"There MUST not be more than one Sparkplug Primary Host Client connected to any MQTT Server"
I cannot agree with that.  

USE CASE 1: 
If I have site A and B, I want to use the same HA MQTT Infrastructure.  All of the Site A EoN Nodes must look for the primary Application for site A, and all of the Site B EoN Nodes must look for the primary application for Site B.

**Comment by wes-johnson** [comment]
Fair enough - this is valid. I'll make sure to address this use case as I work on spec cleanup.
